openapi/SwarmDebug.yaml

```
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 info:
-  version: 1.2.1
+  version: 2.2.1
```

```suggestion
  version: 2.0.0
```

"higher-level" version bump resets the "lower-level" versions ;-)